chain/chain/src/resharding/resharding_actor.rs

```diff
@@ -46,14 +46,7 @@ impl Handler<MemtrieReloadRequest> for ReshardingActor {
 
 impl ReshardingActor {
     pub fn new(store: Store, genesis: &ChainGenesis) -> Self {
-        Self {
-            chain_store: ChainStore::new(
-                store,
-                genesis.height,
-                false,
-                genesis.transaction_validity_period,
-            ),
-        }
+        Self { chain_store: ChainStore::new(store, false, genesis.transaction_validity_period) }
```

Comment:
If you can get rid of the `transaction_validity_period` we could remove genesis all together from here. Just saying ;)

Reply:
I'll try...